auslib/migrate/versions/013_compress_history.py

```diff
@@ -0,0 +1,18 @@
+
+
+def upgrade(migrate_engine):
+    con = migrate_engine.connect()
+
+    if migrate_engine.name == 'mysql':
+        con.execute('ALTER TABLE releases_history ROW_FORMAT=COMPRESSED')
+        con.execute('ALTER TABLE permissions_history ROW_FORMAT=COMPRESSED')
+        con.execute('ALTER TABLE rules_history ROW_FORMAT=COMPRESSED')
```

Comment:
I know it will make it a bit more of a pain to implement, but I think we should explicitly avoid compressing rules or permissions history. There's little to no benefit, so there's no point in making those tables slower.